docs/_theme/djangodocs/static/console-tabs.css

```diff
@@ -0,0 +1,46 @@
+@import url("http://netdna.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css");
```

Comment:
I guess we might want to include a copy of this so that the docs work locally without an internet connection?

Reply:
Good catch. Done.